bindings/rascal/representations/spherical_expansion.py

```diff
@@ -161,7 +161,6 @@ def __init__(
         Required arguments are all the hyperparameters named in the
         class documentation
         """
-
```

Comment:
To make a clean merge, I would remove this change, since the file was not touched otherwise